config_merge.yml

```diff
@@ -0,0 +1,18 @@
+name: 'PRLabeler'
```

Comment:
What's the purpose of this file?
From my understanding, we only need the other two files.

Reply:
actually for the past 1 day i studied about pull request and labels so I saw this file on one repo, so I included it

Reply:
Can you provide me with an example?